intake/cli/client/subcommands/get.py

```diff
@@ -28,7 +29,7 @@
 #-----------------------------------------------------------------------------
 
 class Get(Subcommand):
-    ''' Get a catalog entry
+    ''' Get a catalog entry as CSV
```

Comment:
Oh I see, so this changes the default case even when not writing to a file. I wonder if the original formatted text version isn't also useful, what do you think?

Reply:
I don't think it's useful assuming there is a CSV output, but maybe someone somewhere disagrees? 
 😃

I could refactor to conditionally output csv if you want?